### ISSUE TITLE
Adding Programmer resource pack results in crash

### ISSUE THREAD

**Issue by capSAR273** [baileyholl/Ars-Nouveau#99]
Hello, here's my current setup with this issue:
Pack: Direwolf20 1.16, version 1.0.0
Additional Mods Loaded: FTB Ultimine and Optifine HD_U_G5 (latest for 1.16)

I've commented on the pack's issue page here with the full details: **https://github.com/FTBTeam/FTB-Presents-Direwolf20-1.16/issues/5**

Seems like whether Optifine is loaded or not, the crash still occurs.

**Comment by tantonb** [comment]
I have also been able to reproduce a similar crash whenever I load a resource pack on Forge 35.1.13/35.1.7 with Immersive Engineering and Ars Nouveau 1.8.0+.  The NPE doesn't get thrown in Ars Nouveau 1.7.1 or when Immersive Engineering isn't present.  

See crash report here: [crash report](https://paste.ee/p/vY8R4)

**Comment by Svangen** [comment]
I had the same problem. I tried it in both All the Mods 6 and the DW20 pack. The issue is with Ars Nouveau and Immersive Engineering. The same issue came up when I loaded an instance with just IE and AN and loaded a resource pack. Any resource pack seems to do it. I used Faithful, and a bunch of other ones as well. Oddly enough it worked fine if I disabled AN or IE, loaded the resource pack, and then enabled the mod and restarted the instance. However, I didn't have sound when I did that, but it could have been unrelated.

Here's my crash report from ATM6: https://pastebin.com/4R17tHfU
